### DISCUSSION TITLE
How to remove unused destination account name

### DISCUSSION THREAD

**Discussion by Issam2204** [firefly-iii/firefly-iii#7937] [General]
Hello, first of all thank you for this great piece of software! 

I'm using the importer to integrate my transactions, and for a couple of transactions I did no fill-in correctly the destination account name field. I easily edited those transactions in the UI, but I noticed that the entry is still suggested even if not used by any of my transactions. How do I delete those unnecessary entries? Here is a screenshot:
![image](https://github.com/firefly-iii/firefly-iii/assets/8282518/ad99a59a-2ac5-4f26-aa07-65782afe9190)

Thanks in advance.

**Comment by Issam2204** [comment]
I feel like an idiot, I just found it. It's under Accounts --> Expense accounts.

My apologies :)

**Reply by JC5** [reply]
If you want to delete them all, check out Options > Profile > Data management
